corehq/apps/sso/utils/message_helpers.py

```diff
@@ -19,3 +26,61 @@ def get_success_message_for_trusted_idp(idp, domain_obj):
         domain=domain_obj.name,
         learn_more='#',  # todo include learn more link
     )
+
+
+def render_multiple_strings(context, *templates):
+    """
+    Convenience utility that renders multiple templates with the same context.
+
+    :param context: The context (dict) used to render all `*templates`.
+    :param *templates: One or more template names (strings) to be rendered.
+    :return: Generator of strings -- rendered text of each provided template.
+    """
+    return (render_to_string(template, context) for template in templates)
+
+
+def get_idp_cert_expiration_email(idp):
+    """
+    Utility to generate metadata and render messages for an IdP certificate
```

Comment:
nit: the metadata for django templates we usually refer to as template context ^_^